app/models/chart_service.rb

```diff
@@ -1,85 +0,0 @@
-class ChartService
```

Comment:
So I killed this class and broke it into three: the chart_data_prep which gets the data for the chart and formats it, the chart_renderer which takes the formatted data and returns highchart graph objects, and the overall_chart/comparison_chart services which then put the above two together and return the charts to the dashboards controller to be rendered in the view.  Does the overall breakdown seem clear and maintainable @jmejia?